### ISSUE TITLE
How are we going to handle Icons?

### ISSUE THREAD

**Issue by peelyKoodoo** [KoodooHQ/K2#26]
https://github.com/KoodooHQ/K2/pull/13#discussion_r763298277

**Comment by stevelouiscooper** [comment]
My brain wants to solve this in a similar way to theming.

- Standardised icon set established with a naming convention.
- Can have an icon set per partner or a standard icon set.
- These icons are referenced in some manner by the component and the theme applied dictates which directory or url is referenced.

This approach is agnostic to whether we host the Icons in KML (my preference) or somewhere else. The benefit to doing things inside of KML would be that ideally we could export from Figma -> GitHub in a similar way we plan to do with themes.

**Comment by peelyKoodoo** [comment]
Yup, my opinion is Icons in the repo, not a CDN (So we can source control them, no jump when page loads), and they should just be referenced in the component as names ("backArrow", "clock", "calendar"), and the theme determines which icon that actually is.
